content/scientists.md

```diff
@@ -57,3 +59,10 @@ title: Scientists
 - [Haijiang Zhang](http://staff.ustc.edu.cn/~zhang11/)
 - [Shijie Zhong](http://anquetil.colorado.edu/szhong/)
 - [Lupei Zhu](http://www.eas.slu.edu/People/LZhu/home.html)
+
+## Research Groups
+
+- [Harvard Seismology](http://seismology.harvard.edu/)
+- [Global Seismology Group at Rice University](http://www.gseis.rice.edu/)
+- [Berkeley Global Seismology](http://seismo.berkeley.edu/wiki_br/Main_Page)
+- [University of Rhode Island Seismology Lab](https://sites.google.com/view/seismo)
```

Comment:
I don't think we should maintain the list of research groups.

Reply:
Remove in d5fac44.